README.md

```diff
@@ -40,6 +40,9 @@ To get started, you have to install the dependencies (mainly snowpack)
 npm install
 ```
 
+### Environment Labels
```

Comment:
There's a slight change to the setup instructions in master, and I wonder if this actually belongs in there. If you merge master in, you could add a step, like:
```diff
diff --git a/README.md b/README.md
index cfee13c..1b80c18 100644
--- a/README.md
+++ b/README.md
@@ -35,14 +35,12 @@ This started as a very lightweight, single page html file, and we've tried very
 ### Setup
 
 1. Install node at the version specified in the [`.node_version`](.node_version) file. If you use a version manager like nodenv or nvm, this should be detected automatically.
-2. Install dependencies with npm
+2. Create a `.env` file with secrets needed to run the app. You can get the secrets you need from someone in the Open Twin Cities slack `#tc-aid-dev` channel.
+3. Install dependencies with npm
     ```
     npm install
     ```
-3. Start building!
-
-### Environment Labels
-In order to run locally you will need to retrieve the `.env` file from someone in the Open Twin Cities slack `#tc-aid-dev` channel.
+4. Start building!
 
 ### Run locally
```

Reply:
sounds fine to me is there a way we can just commit this suggestion?

Reply:
If you merge master into the branch I can convert it from a diff to a suggestion

Reply:
done

Reply:
I think we can tidy up the readme instructions after merging this in. No point in going back and forth with more review